### PR TITLE
Red Icon Flicker Issue Resolved

### PR DESCRIPTION
**Pull request by arvind-source**
#### How to test

Red Icon Flicker Issue on  "No known contacts" page issue resolved

### CODE REVIEW

**Comment by arvind-source** [comment]
Red Icon Flicker Issue I have resolved

File: android/build.gradle [hunk 1]
```diff
@@ -18,7 +18,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath("com.android.tools.build:gradle:3.4.2")
+        classpath('com.android.tools.build:gradle:3.6.3')
```
Comment (Patrick-Erichsen) [inline android/build.gradle:21]: Please remove this upgrade

File: android/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,5 +1,6 @@
+#Thu Apr 30 12:05:19 IST 2020
```
Comment (Patrick-Erichsen) [inline android/gradle/wrapper/gradle-wrapper.properties:1]: Please remove this line as well as line 6

**Comment by Patrick-Erichsen** [comment]
Hey @arvind-source, thanks for contributing!

Could you provide some instructions on how to verify that this change is working as intended?

It looks like the Andoird build is failing as well. I'm assuming this is related to the gradle version bump that is in this PR - please remove that logic.

File: android/build.gradle (hunk 1 above)
Comment (arvind-source) [reply android/build.gradle:21]: Okay sure @Patrick-Erichsen

**Pull request closed without merging**